screens/historyHelpers.js

```diff
@@ -0,0 +1,41 @@
+import BASE from '../lib/common';
+
+const getCustomerTransactions = function async(userId) {
```

Comment:
This is really good! I realized while reviewing your code that there's a better way to interact with Airtable -

Refactor to use filterByFormula, and a sort function with the date! This way we don't have to iterate over _all_ transactions, and we only make a couple API calls versus other methods. And we can take advantage of Airtable built-in things like the sort function (":

Couple other code cleanup things:
- Use Promise.all  --> .then --> .map to create an array; use a helper function called 'createTransactionData' or something like that
- Here's an example from Tommy's PR (which I also refactored 😂)

```
      Promise.all(productRecords)
        .then(records => {
            const products = records.map(record => createProductData(record));
            resolve(products);
        })
    .catch(err => reject(err));
```

Reply:
Wow it really is much shorter now